10_deploy_rook.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/bash
+
+git clone https://github.com/rook/rook
+cd rook/cluster/examples/kubernetes/ceph
+oc create namespace rook-ceph-system
+oc create configmap csi-cephfs-config -n rook-ceph-system --from-file=csi/template/cephfs
+oc create configmap csi-rbd-config -n rook-ceph-system --from-file=csi/template/rbd
+oc apply -f csi/rbac/rbd
+oc apply -f csi/rbac/cephfs
+sed -i '/ROOK_HOSTPATH_REQUIRES_PRIVILEGED/!b;n;c\          value: "true"' operator-with-csi.yaml
+oc create -f scc.yaml
+oc adm policy add-scc-to-user privileged -z rook-csi-rbd-provisioner-sa -n rook-ceph-system
+oc adm policy add-scc-to-user privileged -z rook-csi-rbd-attacher-sa -n rook-ceph-system
+oc adm policy add-scc-to-user privileged -z csi-rook-csi-rbd-plugin-sa -n rook-ceph-system
+oc adm policy add-scc-to-user privileged -z rook-csi-cephfs-plugin-sa -n rook-ceph-system
+oc adm policy add-scc-to-user privileged -z rook-csi-cephfs-provisioner-sa -n rook-ceph-system
+oc create -f operator-with-csi.yaml
+sleep 180
+sed -i "s/useAllDevices: .*/useAllDevices: true/" cluster.yaml
+oc create -f cluster.yaml
+oc create -f toolbox.yaml
+cat <<EOF | oc create -f -
+echo apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: rbd
+  namespace: rook-ceph
+spec:
+  failureDomain: osd
+  replicated:
+    size: 3
+EOF
+
+sleep 240
```

Comment:
Same as above, for CI we want to wait for as little time as possible, and some poll/event check will be generally more robust?

See #130 for example where we started using oc wait to ensure a given condition or event happened?